### PR TITLE
Refactor a bit to consolidate less modules

### PR DESCRIPTION
**Pull request by wjramos**

### CODE REVIEW

**Comment by mhewson** [comment]
looking good I would like to run a visual regression prior to taking these into master however

**Comment by wjramos** [comment]
Yup, planning on it --

It should reduce file size of source and distributable files without affecting actual computed styles

File: src/less/definitions/collections/forms.less [hunk 1]
```diff
@@ -114,97 +116,107 @@ label {
...
     }
-
     &[disabled],
     fieldset[disabled] & {
         cursor: @cursor-disabled;
     }
-
     // Reset height for `textarea`s
-    textarea& {
+    textarea & {
```
Comment (wjramos) [inline src/less/definitions/collections/forms.less]: I'll need to confirm this is correct

File: src/less/definitions/collections/forms.less [hunk 2]
```diff
@@ -114,97 +116,107 @@ label {
...
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-//
-// Validation messages
-//
-// extends alert
-//
-//----------------------------------------------------------------------------------------------------------------------
-.form-control {
-    .rei-form-control-focus();
+    .input-lg + .form-control-feedback {
```
Comment (wjramos) [inline src/less/definitions/collections/forms.less]: Looks like this can be simplified as well

**Pull request closed without merging**